<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioEllipse">
       <property name="text">
        <string>Эллипс</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="radioSquare">
       <property name="text">
        <string>Квадрат</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="radioRhomb">
       <property name="text">
        <string>Ромб</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioRect">
       <property name="text">
        <string>Прямоугольник</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioStar6">
       <property name="text">
        <string>Звезда-6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioStar8">
       <property name="text">
        <string>Зезда-8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioStar5">
       <property name="text">
        <string>Звезда-5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioTriangle">
       <property name="text">
        <string>Треугольник</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioHex">
       <property name="text">
        <string>Шестиугольник</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>368</width>
      <height>278</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushLeft">
       <property name="text">
        <string>Влево</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushRight">
       <property name="text">
        <string>Вправо</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushDown">
       <property name="text">
        <string>Вниз</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushUp">
       <property name="text">
        <string>Вверх</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="whatsThis">
        <string comment="угол поворота"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushCanvas">
       <property name="text">
        <string>Канва</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushScale">
       <property name="text">
        <string>Масштаб</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineScale"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineX"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineY"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>290</y>
      <width>346</width>
      <height>451</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Периметр</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Площадь</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Центр масс</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Точка8</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Значения</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>4</pointsize>
      </font>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
